linux_os/guide/system/ocp_idp_no_htpasswd/oval/shared.xml

```diff
@@ -0,0 +1,33 @@
+{{% set YAML_TEST_OVAL_VERSION = [5, 11] %}}
```

Comment:
Shouldn't this be 5.11.3?
To align with, https://github.com/OpenSCAP/openscap/pull/1482

Reply:
We don't operate with "patch" versions of the OVAL content. The oscap validation is performed on the datastream-level, and the datastream XSD decides what OVAL schema to use for validation (see also https://github.com/OpenSCAP/openscap/pull/1500)

Reply:
I see, but doesn't that mean we will start generating invalid content by default?

Reply:
The YAML test qualifies as an [unofficial OVAL extension](https://oval-community-guidelines.readthedocs.io/en/latest/proposal-process/release-process.html#how-to-release-streams). We can say with high certainty that it will take a lot of time before it is part of the official OVAL - the process works like this: unofficial -> development -> stable -> official.

Reply:
@matejak So do I understand it well that you want to build, release and ship only invalid content? Or will it be a special option that OCP developers have to turn on manually?

Reply:
So now, the rule is not ocp4-only, and as the choice is between incomplete content and content with unofficial extensions, let's go for the latter.